src/test/ui/binding/const-param.stderr

```diff
@@ -0,0 +1,17 @@
+warning: the feature `const_generics` is incomplete and may cause the compiler to crash
+  --> $DIR/const-param.rs:3:12
+   |
+LL | #![feature(const_generics)]
+   |            ^^^^^^^^^^^^^^
+   |
+   = note: `#[warn(incomplete_features)]` on by default
+
+error[E0080]: runtime values cannot be referenced in patterns
```

Comment:
Hmm, so I would expect us to add a case to `report_inlining_errors` with `PatternError::ConstParamInPattern` and have a wording "const parameters cannot be referenced in patterns". (If you feel like it, perhaps add a rationale for curious users who ask "But why not?".)

Reply:
Ok, will add.
(I initially went with "const eval is not my business, so I don't change it".)

Reply:
Done.